.github/workflows/pr.yml

```diff
@@ -74,9 +74,9 @@ jobs:
           APPLE_API_KEY: ${{ secrets.APPLE_API_KEY }}
           APPLE_API_ISSUER: ${{ secrets.APPLE_API_ISSUER }}
           APPLE_API_KEY_PATH: ${{ env.APPLE_API_KEY_PATH }}
-        run: npm run make --arch=arm64,x64
+        run: npm run make
         shell: bash
-  linux:
+  linux-amd64:
```

Comment:
We should support arm64 on Linux too for RPis and such

Reply:
Good point, I cleaned up the platform/arch bits and added linux arm64.